docs/getstarted/last_page.md

```diff
@@ -30,11 +30,11 @@ Depending on your interest, the Docker documentation contains a wealth of inform
   </tr>
   <tr>
     <td class="tg-031e">More about Docker for Mac, features, examples, FAQs, relationship to Docker Machine and Docker Toolbox, and how this fits in the Docker ecosystem</td>
-    <td class="tg-031e">[Getting Started with Docker for Mac](/docker-for-mac/index.md)</td>
+    <td class="tg-031e">[Getting Started with Docker for Mac](/mac/index.md)</td>
   </tr>
   <tr>
-    <td class="tg-031e">More about Docker for Windows, More about Docker for Windows, features, examples, FAQs, relationship to Docker Machine and Docker Toolbox, and how this fits in the Docker ecosystem</td>
-    <td class="tg-031e">[Getting Started with Docker for Windows](/docker-for-mac/index.md)</td>
+    <td class="tg-031e">More about Docker for Windows, features, examples, FAQs, relationship to Docker Machine and Docker Toolbox, and how this fits in the Docker ecosystem</td>
+    <td class="tg-031e">[Getting Started with Docker for Windows](/windows/index.md)</td>
```

Comment:
no, this needs to be `/docker-for-mac` - the checkers don't know about the not-yet-released paths, and will fail.